### PR TITLE
TASK: Add support for pkpass mime type

### PR DESCRIPTION
**Pull request by albe**
Updates the script to automatically update the mime-types to also include pkpass (iOS Wallet) and provides a unique place to add other formats if needed.

### CODE REVIEW

**Comment by dfeyer** [comment]
what about a contribution upstream in the Apache SVN ? We use them as official source for our mime type, can make sense to play the opensource game and contribute upstream.

File: Neos.Utility.MediaTypes/Scripts/updatemediatypes.php [hunk 1]
```diff
@@ -31,6 +31,9 @@
     }
 }
 
+// Add non-registered mime types that we still want to support:
+$mediaTypesAndFileExtensions['application/vnd.apple.pkpass'] = ['pkpass'];
+
```
Comment (aertmann) [inline Neos.Utility.MediaTypes/Scripts/updatemediatypes.php]: is this script run automatically or should it be done manually and included with this PR?

File: Neos.Utility.MediaTypes/Scripts/updatemediatypes.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 $mediaTypesClassPathAndFilename = __DIR__ . '/../Classes/MediaTypes.php';
 
-$rawList = file_get_contents('http://svn.apache.org/viewvc/httpd/httpd/branches/2.4.x/docs/conf/mime.types?revision=HEAD&view=co');
+$rawList = file_get_contents('https://svn.apache.org/viewvc/httpd/httpd/branches/2.4.x/docs/conf/mime.types?revision=HEAD&view=co');
```
Comment (dfeyer) [inline Neos.Utility.MediaTypes/Scripts/updatemediatypes.php]: not sure if the trunk is more up to date ?

File: Neos.Utility.MediaTypes/Scripts/updatemediatypes.php (hunk 1 above)
Comment (dfeyer) [reply Neos.Utility.MediaTypes/Scripts/updatemediatypes.php]: I see nothing on Jenkins

File: Neos.Utility.MediaTypes/Scripts/updatemediatypes.php (hunk 2 above)
Comment (albe) [reply Neos.Utility.MediaTypes/Scripts/updatemediatypes.php]: Was thinking about this too, but apache is slowly going with versions and I'm not sure about their branching policy. Decided to stay with 2.4.x but can be changed ofc.

**Comment by albe** [comment]
Someone needs to run the script once and create a new PR. Not at my computer at the moment, otherwise I'd do it right now.

**Pull request merged**